Documentation/building/cross-building.md

```diff
@@ -1,7 +1,23 @@
-Cross Compilation for ARM
-=========================
+Cross Compilation for ARM on Linux
+==================================
+
+Through cross compilation, on Linux it is possible to build CoreCLR for arm or arm64.
```

Comment:
`ARM` is the official processor/architecture name; `arm`, `arm64`, `amd64`, ... are the CoreCLR build architecture names. I think it is fine to use the lower case here - it implies that it is talking about CoreCLR build architectures.